.github/workflows/Tests.yml

```
@@ -6,6 +6,24 @@ on:
       - stable
     tags:
       - v*
+  pull_request:
+    types:
+      - opened
+      - edited
+      - reopened
+      - ready_for_review
+    branches-ignore:
+      - '*-patc*'
+  pull_request_target:
+    types:
+      - opened
+      - edited
+      - reopened
+      - synchronize
+      - ready_for_review
+    branches:
+      - master
+      - stable
```

🤔 By expanding when `MATS` are checked this may have been the wrong approach.

This change to something simpler like
```suggestion
  pull_request:
    types:
      - opened
      - edited
      - reopened
      - synchronize
      - ready_for_review
```

---

:+1: Resolved by:
 - https://github.com/reactive-firewall/python-repo/pull/42/commits/bca55eb4de46b3f192e470ec7debe3bb721051df#diff-f88e7ee60a80885f57e6c7794c2dd3d3604ffed08c7c2bde856b1669186e8444L10-L17
 - and https://github.com/reactive-firewall/python-repo/pull/42/commits/bca55eb4de46b3f192e470ec7debe3bb721051df#diff-f88e7ee60a80885f57e6c7794c2dd3d3604ffed08c7c2bde856b1669186e8444L24-L26